Index: include\gbs_types.h
===================================================================
--- include\gbs_types.h
+++ include\gbs_types.h
@@ -59,8 +59,15 @@
 
     // Collisions
     uint8_t collision_group;
 
+    //Dynamic actor
+    uint8_t actor_behavior_id;
+    uint8_t actor_state;
+    int16_t actor_vel_x;
+    int16_t actor_vel_y;
+    uint8_t actor_linked_actor_idx;
+
     // Linked list
     struct actor_t *next;
     struct actor_t *prev;
 } actor_t;
